websites/W/Wolvesville/presence.ts

```diff
@@ -124,14 +137,14 @@ presence.on("UpdateData", async () => {
     if (menu) {
       data.details = "At the main menu";
 
-      if (showUsername) {
+      if (privacyMode === false) {
```

Comment:
```suggestion
      if (!privacyMode) {
```

Reply:
That's how I had them in the beginning but it didn't work properly (spent like an hour trying to figure out why). I'll try testing it again though.

Reply:
Nvm works now, idk what was wrong with it when I tested it some weeks ago